plugins/postfix/postfix_stats

```diff
@@ -110,7 +121,11 @@ fi
 
 # Variable to store the pflogsumm result.
 # shellcheck disable=SC2086
```

Comment:
Please move this shellcheck attribute down to the line it belongs to. See the output of `shellcheck -s dash postfix_stats`.